src/Namotion.Reflection/XmlDocsExtensions.cs

```diff
@@ -52,66 +52,73 @@ internal static void ClearCache()
         /// <summary>Returns the contents of the "summary" XML documentation tag for the specified member.</summary>
         /// <param name="type">The type.</param>
         /// <param name="resolveExternalXmlDocs">Specifies whether tho resolve the XML Docs from the NuGet cache or .NET SDK directory.</param>
+        /// <param name="formattingMode">Specified how formatting tags should be processed.</param>
         /// <returns>The contents of the "summary" tag for the member.</returns>
-        public static string GetXmlDocsSummary(this CachedType type, bool resolveExternalXmlDocs = true)
+        public static string GetXmlDocsSummary(this CachedType type, bool resolveExternalXmlDocs = true, XmlDocsFormattingMode formattingMode = XmlDocsFormattingMode.Unformatted)
```

Comment:
I agree with @RicoSuter closing to threshold to introduce `XmlDocOptions` or similar, especially as these parameters control so many methods

Reply:
Added `XmlDocOptions` that contains `ResolveExternalXmlDocs` and `FormattingMode` and can be easily extended.
For backward and binary compatibility, previous versions of all methods are restored (which means, parameter `formattingMode` was removed).

Reply:
As the static class is called XmlDocsExtensions, should this option be called XmlDocsOptions?
If we align it with NJS/NSwag it should even be XmlDocSettings, however .NET prefers options for these classes, right?

Reply:
I did some research and saw, that the postfixes `Settings` and `Options` seem to be used quite interchangably.
In one discussion on Stackoverflow one said, that `Settings` are more like system- or applicationwide values, while `Options` are used to apply variations to that settings in a component scope.
But this also seems to be his personal opinion as I couldn't find a more "official" document on this.

Back to our case ... I think we can decide freely what to use and I understand your wish to have consistent naming in your projects. Thus `Settings` seems to be the way to go.

If you agree, I'll rename the class!

Greetings from Austria!